test/verify/check-packagekit

```diff
@@ -231,6 +240,179 @@ class TestUpdates(NoSubManCase):
+                        b.wait_not_present("#restart-services-modal")
+
+                # test the tracer functionality works in-page after sucessfull update
+                else:
+                    if self.rebootRequired:
+                        # update required a reboot
+                        b.wait_not_present("#choose-service")
+                        b.click("#reboot-system")
+                        b.wait_visible("#shutdown-dialog")
+                        b.click("#delay")
+                        b.click("button:contains('No delay')")
+                        b.wait_text("#delay .pf-c-select__toggle-text", "No delay")
+                        b.click("#shutdown-dialog button:contains('Restart')")
+                        b.switch_to_top()
+                        b.wait_in_text(".curtains-ct h1", "Disconnected")
+
+                        # ensure that rebooting actually worked
+                        m.wait_reboot()
+                        m.start_cockpit()
+                        b.reload()
+                        b.login_and_go("/updates")
+                    else:
+                        # update required a service restart
+                        b.wait_not_present("#reboot-system")
+                        b.click("#choose-service")
+                        b.wait_visible("#restart-services-modal")
+                        b.wait_in_text(".restart-services-modal-body", self.serviceName)
+                        b.click("#restart-services-modal button:contains('Restart services')")
+                        b.wait_not_present("#restart-services-modal")
+
+                # check no updates are present
+                b.wait_in_text("#state", "System is up to date")
+
+                # history on "up to date" page should show the recent update (expanded by default)
+                self.testObj.assertHistory("#expanded-content1 > div > ul", [self.packageName])
+
+            def verify_backend(self):
+                if not self.rebootRequired:
+                    # Check the service was actually restarted
+                    newServiceStartTime = m.execute("systemctl show {0}.service --property=ExecMainStartTimestamp".format(self.serviceName))
+                    self.testObj.assertTrue(self.serviceStartTime != newServiceStartTime)
+
+                # check no services/processes need reboot or service restart
+                # tracer returns non-zero exit code if any services/processes are affected
+                m.execute("tracer --all --root")
+
+            def cleanup(self):
+                m.execute("rm /usr/local/bin/{0}".format(self.appName))
+                m.execute("systemctl stop {0}.service".format(self.serviceName))
+                m.execute("rm /etc/systemd/system/{0}.service".format(self.serviceName))
+                if self.rebootRequired:
+                    m.execute("rm /etc/tracer/applications.xml")
+
+
+        if m.execute("which tracer || true") == "":
+            return
```

Comment:
Please replace this with a `@skipImage()` decorator. This will (1) make sure that our fedora images don't regress by accidentally dropping tracer again, (2) alert us to fix new images if they lack tracer (e.g. rhel-9-0), and (3) avoid needlessly booting a VM for an empty test.

The skipped OS list will be rather long right now for sure, but over time it will shrink.